components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -49,8 +57,34 @@ export const AppSideBar = () => {
     .sort((a, b) => new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime());
 
   return (
-    <div className="pt-12 bg-white flex flex-col gap-8 justify-between min-w-max min-h-full">
+    <div className="fixed top-0 pt-12 bg-white flex flex-col gap-8 justify-between min-w-max min-h-full">
       <div className="grid gap-4 mt-4 px-4">
+        <label className="flex flex-col gap-2 w-max">
+          <span className="sr-only">Workspace</span>
+          <ClientOnly>
+            <SingleSelect
```

Comment:
The values here don't update at the moment when someone edits the workspace settings. I'm looking to get this sorted today.

Reply:
Fixed in e48438b839370268bd7cf0f4b4fd4e39fd20d2a0